operators/bosh-restore-operator/BoshRestoreService.js

```diff
@@ -90,7 +90,7 @@ class BoshRestoreService extends BaseDirectorService {
         resourceId: opts.restore_guid,
         options: optionsData,
         status: {
-          'state': `${CONST.APISERVER.RESOURCE_STATE.IN_PROGRESS}_BOSH_STOP`,
+          'state': `${CONST.APISERVER.RESOURCE_STATE.TRIGGER}_BOSH_STOP`,
           'response': data
         }
       });
```

Comment:
Is this catch really necessary? There is a catch already present in [DefaultBoshRestoreOperator](https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/648/files#diff-f12ac58b1cf691827c26f10a44281436L37)

Reply:
yep, this `try...catch` can be removed.

Reply:
Done